firstvoices/backend/serializers/site_data_serializers.py

```diff
@@ -122,11 +122,10 @@ def get_optional(self, dictionaryentry):
                 ),
                 **(
                     {
-                        "Part of Speech": dictionaryentry.translation_set.first().part_of_speech.title,
+                        "Part of Speech": dictionaryentry.part_of_speech.title,
                     }
                     if dictionaryentry.translation_set.first() is not None
```

Comment:
Thanks for updating this serializer. I don't think we need to check the translation_set at all here anymore